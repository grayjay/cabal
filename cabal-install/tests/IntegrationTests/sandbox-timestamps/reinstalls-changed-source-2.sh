. ./common.sh

cd p
cabal sandbox init > /dev/null

cabal sandbox add-source ../q > /dev/null

cabal install --only-dependencies > /dev/null
cabal run p -v0
echo '       ++ " edited"' >> ../q/Q.hs
cabal run p -v0
