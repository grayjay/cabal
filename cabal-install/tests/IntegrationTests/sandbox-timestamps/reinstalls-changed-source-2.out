message
In order, the following will be installed:
q-1.0 (reinstall)
message edited
